visualize: True
dlio:
  estimated_file: "_dlio_evo.tum"
  reference_file: "_cpt7_ie_tc_evo.tum"
  parameters:
    t_max_diff: 0.02
    t_offset: 0.0
    n_to_align: -1
    correct_scale: False

o3d:
  estimated_file: "open3d_slam_replayed_evo.tum"
  reference_file: "_cpt7_ie_tc_evo.tum"
  parameters:
    t_max_diff: 0.02
    t_offset: 0.0
    n_to_align: -1
    correct_scale: False

se:
  estimated_file: "_lpc_state_estimator_evo.tum"
  reference_file: "_cpt7_ie_tc_evo.tum"
  parameters:
    t_max_diff: 0.01
    t_offset: 0.0
    n_to_align: -1
    correct_scale: False